src/language-server/definition-finder.ts

```diff
@@ -12,14 +12,17 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
+import * as fuzzAldrin from 'fuzzaldrin';
```

Comment:
curiosity: why capital A in fuzzAldrin here? You're using filter only, wdyt of importing only that?

Reply:
`filter` is too vague of a word for what it's doing, and since we're only calling it one place it seemed easier to read as `fuzzAldrin.filter`. Renamed to lowercase to match the package name.